setup.py

```diff
@@ -48,6 +48,15 @@ def read_file(filename, encoding='utf8'):
     'zope.interface',
 ]
 
+# Add pywin32 on Windows platform to handle low-level system calls.
+# This dependency needs to be added dynamically to avoid to be here
+# when certbot-oldest tests are launched.
+# Indeed, associated with --constraint, pip will ignore platform_system directive and try
```

Comment:
What do you mean "associated with --constraint"?

Reply:
pyYes it is not obvious from this comment. When `pip_install.py` executes to build a oldest environment, it will invoke pip using a specific constraint file specified by `--constraint`. This as a consequence to ignore every environment marker, and try to install `pywin32` on a Unix platform.

This code fix that.

Reply:
I refactored a little the comment.